tests/features/support/steps_clone.js

```diff
@@ -17,173 +18,173 @@ KEY:
 
 
 // =========================
 // Navigational
-// =========================
+// ========================= 
 //x @step(r'I exit by taping "([^"]+)"')
 //x @step(r'If I see it, I will tap the link "([^"]+)"')
 
-// ? I tap the ${ specified } link "([^"]+)"
+// ? I tap the ${ specified } link "([^"]+)" 
 
 // Also sometimes not navigational
-// ^ @step(r'I tap the button "([^"]+)"')
+// ^ @step(r'I tap the button "([^"]+)"') 
 // ? I tap "([^"]+)" (can this be a button or link or anything?) (Include 1st, 2nd, etc?)
 // ? @step(r'I tap the "([^"]+)" button')
-// ? @step(r'I tap the link "([^"]+)"')
+// ? @step(r'I tap the link "([^"]+)"') 
 // Maybe go with 'I tap'? Is that possible? Maybe we should stick to the script.
-// √ I tap the (button|link) "([^"]+)"
+// √ I tap the (button|link) "([^"]+)" 
 
-// √ the link "([^"]+)" should open a working page
+// √ the link "([^"]+)" should open a working page 
 
 
 // =========================
 // Interactive
 // =========================
-//x @step(r'I upload the file "([^"]*)"')
+//x @step(r'I upload the file "([^"]*)"') 
 //x @step(r'I reload the screen')
-//x @step(r'I select "([^"]+)" from the menu')
+//x @step(r'I select "([^"]+)" from the menu') 
 //x @step(r'I tap the back button')
 //x @step(r'I tap the question back button')
-//x @step(r'I tap the (first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth) link "([^"]+)"')
+//x @step(r'I tap the (first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth) link "([^"]+)"') 
 //x @step(r'I tap the final link "([^"]+)"')
 
-// ? @step(r'I select "([^"]+)" in the combobox')
+// ? @step(r'I select "([^"]+)" in the combobox') 
 // ~ I pick "([^"]+)" in the { specified } combobox
 
 // ? @step(r'I set the combobox text to "([^"]*)"')
-// ~ I set the {specified} combobox text to "([^"]*)"
+// ~ I set the {specified} combobox text to "([^"]*)" 
 
 //x too vague @step(r'I choose "([^"]+)"')  // dropdown
 // ^ I select "" from the "" dropdown
 // ? I choose "([^"]+)" from the ${ descriptor } dropdown
 // ? I set the "([^"]*)" dropdown to "([^"]*)"
-// ? I pick "" from the "" dropdown?
+// ? I pick "" from the "" dropdown? 
 
 // These could be better differentiated
-// ? @step(r'I select "([^"]+)" as the "([^"]+)"')  // options
+// ? @step(r'I select "([^"]+)" as the "([^"]+)"')  // options 
 // ? @step(r'I select "([^"]+)" as the (first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth) "([^"]+)"')
 // All could be combined with regex?
-// ? I pick ""?
+// ? I pick ""? 
 
 // They're both (all?) text fields. Combine these two? The end behavior is a little different for some reason.
-// 'set' is a good way of expressing that they're cleared first, but maybe clearing should be done separately?
+// 'set' is a good way of expressing that they're cleared first, but maybe clearing should be done separately? 
 // Or maybe there should be 'set', 'type', and 'clear' options.
```

Comment:
We don't have to change this now, but I think "fill" sounds more like spoken English than "set," and to me, at least, has the same connotation of clear and then type. The sentence would looks something like "I fill the ___ textbox/textarea with ___"

Reply:
I don't think I'd thought of `fill`! If we know we want to change it, we should change it now I think, before a bunch of tests are written differently.

Reply:
I think I added two 'fill' options in there.